datasets/told-br/told-br.py

```diff
@@ -0,0 +1,178 @@
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="multilabel",
+            version=VERSION,
+            description="""
+            Full multilabel dataset with target values ranging
+            from 0 to 3 representing the votes from each annotator.
+            """,
+        ),
+        datasets.BuilderConfig(
+            name="binary",
+            version=VERSION,
+            description="""
+            Binary classification dataset version separated in train, dev and test test.
+            A text is considered toxic if at least one of the multilabel classes were labeled
+            by at least one annotator.
+            """,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "binary"
+
+    def _info(self):
+        if self.config.name == "binary":
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "toxic": datasets.Value("uint32"),
+                }
+            )
+        else:
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "homophobia": datasets.Value("uint32"),
+                    "obscene": datasets.Value("uint32"),
+                    "insult": datasets.Value("uint32"),
+                    "racism": datasets.Value("uint32"),
+                    "misogyny": datasets.Value("uint32"),
+                    "xenophobia": datasets.Value("uint32"),
+                }
+            )
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION, features=features, homepage=_HOMEPAGE, license=_LICENSE, citation=_CITATION
+        )
+
+    def _split_generators(self, dl_manager):
+        urls = _URLS[self.config.name]
+        data_dir = dl_manager.download_and_extract(urls)
+        if self.config.name == "binary":
+            return [
+                datasets.SplitGenerator(
+                    name=datasets.Split.TRAIN,
+                    gen_kwargs={"filepath": os.path.join(data_dir, "1annotator/ptbr_train_1annotator.csv")},
+                ),
+                datasets.SplitGenerator(
+                    name=datasets.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(data_dir, "1annotator/ptbr_test_1annotator.csv")},
+                ),
+                datasets.SplitGenerator(
+                    name=datasets.Split.VALIDATION,
+                    gen_kwargs={"filepath": os.path.join(data_dir, "1annotator/ptbr_validation_1annotator.csv")},
+                ),
+            ]
+        else:
+            return [
+                datasets.SplitGenerator(
+                    name=datasets.Split.TRAIN,
+                    gen_kwargs={
+                        "filepath": os.path.join(data_dir),
+                    },
+                )
+            ]
+
+    def _generate_examples(self, filepath):
+        df = read_csv(filepath, engine="python")
```

Comment:
(nit) can you change the import to `import pandas as pd` and do this please ?
```suggestion
        df = pd.read_csv(filepath, engine="python")
```

This is for consistency and to avoid data streaming issues (we extend pandas for streaming when it's imported this way)